### ISSUE TITLE
Translate bitcoin threat model to portugese

### ISSUE THREAD

**Issue by JWWeatherman** [JWWeatherman/math_bot#146]

**Comment by JWWeatherman** [comment]
need to assign to damujen once he accepts the collaborator invite.

**Comment by JWWeatherman** [comment]
@dy24703 can you link to the translation on this bug?

**Comment by dy24703** [comment]
You want the link posted here?

**Comment by JWWeatherman** [comment]
Yes, please.

**Comment by dy24703** [comment]
[Vision Statement for Mathbot pt-br.docx](https://github.com/JWWeatherman/math_bot/files/2308088/Vision.Statement.for.Mathbot.pt-br.docx)

**Comment by dy24703** [comment]
He posted it as a word document, so you will have to download that. I can maybe copy it to a google docs if that is preferable?

**Comment by dy24703** [comment]
Shit I just realized I posted the vision statement translation. I think he is still working on this one.

**Comment by JWWeatherman** [comment]
No, that’s fine. Just upload it to google drive and share it as is.

**Comment by JWWeatherman** [comment]
Cool.
